manifests/profiles/PROFILES.md

```diff
@@ -10,7 +10,7 @@ will provide further instructions. Additionally, please read the RFC requirement
 
 ## Vendor optimized profiles
 
-| K8s Platform         | Vendor        | Lead Maintainer | Lead Maintainer  |
-| -------------------- | ------------- | --------------- | ---------------- |
-| OpenShift            | Red Hat / IBM | @jwendell       | @irisdingbj      |
-| Upstream Kubernetes  | Istio         | Env WG          | discuss.istio.io |
+|Profile Name| K8s Platform         | Vendor        | Lead Maintainer | Lead Maintainer  |
+|------------| -------------------- | ------------- | --------------- | ---------------- |
```

Comment:
I think a space is needed between `|` and `-` such that the table renders properly. Double check the output in your local github fork.